### ISSUE TITLE
Link: Usage tab

### ISSUE THREAD

**Issue by thyhmdo** [carbon-design-system/carbon-website#3537]
**Ref discussion:** https://github.com/carbon-design-system/carbon/discussions/15175
**Ref boxnote:**[ Usage tab pdf](https://ibm.box.com/s/p3i0fndg29pb8l5wvnd1l953aet27q0s)

 ## Usage tab

- [x] **Description:** Possibly reword this sentence if needed to describe different use cases. Should we cover those here? Are we covering all of them currently?
- [x] **Overview section:** When not to use: Reword this paragraph to be more clear. Gower has commented on this in his [PDF](https://ibm.ent.box.com/file/969036329755?s=lllsvdsz7kpxw12j9lxdewyg5bclazse).
- [x] **Anatomy section:** In image: Change "Learn more" to something like "View docs".
<img width="649" alt="Screen Shot 2023-05-15 at 4 51 34 PM" src="https://github.com/carbon-design-system/carbon-website/assets/43969356/45628f3f-9667-427d-93d8-e1cfe9ea53ec">

- [x] **Content:**
- Clarify that we `recommend ` labels accurately reflect the content users will find at the link destination. This may not always be possible.
- "For further content guidance, see IBM Accessibility on link purpose." Revise this to link to the [Accessibility tab](https://carbondesignsystem.com/components/link/accessibility) for more information

- [x] **Interactions section:** Remove Screen reader parts.

- [x] **Standalone link section:** 
- Change sentence to this "Standalone links are the default link style for Carbon and only have an underline on hover and focus states."
- Change the image to have different link content. For example, say something like "View docs" with a right arrow icon.
<img width="594" alt="Screen Shot 2023-05-15 at 5 06 50 PM" src="https://github.com/carbon-design-system/carbon-website/assets/43969356/fc455ebe-46d4-4344-ab40-3e54c2d8239f">

- [x] **Visited style section:** Is it an accessibility violation to not use visited link styles? If you are not using visited styles, is there a known accessibility consideration that could be put in place? Ask Gower.

- [x] Add somewhere what our stance is on not allowing full images as pure links, but that it is okay to have an image within a card + text as a link, etc.

----

### Answered questions from a11y discussion

**Is a link made up of text + icon really a link, or is it a button?**
A link is a link, not a button, regardless if it is just text or text + icon. We need to be clearer about the content we use as examples in our react storybook and website guidance to establish this.

**In-page anchor links are violating the guidance for link color and icon placement.**
The anchor links on the Carbon website are coming from Gatsby, not our core components. That being said, we should at least provide an underline on hover and focus in Gatsby. This will be a separate issue.

**What about links that are just images?**
We do not recommend using pure images as links. We have seen use cases of having cards that also include blocks on content within the container below the image treated as a link. We can add more about this in our guidance.

**Comment by oliviaflory** [comment]
### Examples of link + icon in IBM.com

#### CTA component
Within the `CTA` component in Carbon for Dotcom, we pull in the link with icon and pre-populate the following options:
[Storybook link](https://www.ibm.com/standards/carbon/web-components/?path=/story/components-cta--default)

- Local: arrow-right
- Anchor: arrow-down
- External: launch
- New tab: new-tab
- Download: download
- PDF: document-pdf
- Blog: blog
- Email: email
- Call: phone
- Schedule: calendar
- Chat: chat

Outside of local, anchor, and external, the above have been requested from adopters to utilize in various pages.


--------
### Live examples
[What is AI](https://www.ibm.com/artificial-intelligence?lnk=flathl)

<img width="1693" alt="Screenshot 2023-05-15 at 3 50 28 PM" src="https://github.com/carbon-design-system/carbon-website/assets/45692622/f57c2a94-a1ed-4cc9-b84d-e0145e80fe1c">



[API Connect](https://www.ibm.com/products/api-connect/resources)

<img width="1694" alt="Screenshot 2023-05-15 at 3 53 56 PM" src="https://github.com/carbon-design-system/carbon-website/assets/45692622/9a207b2a-82a1-4c26-949a-5f5946536509">

- Note this example will likely transition to utilize the updated card specs

<img width="1307" alt="Screenshot 2023-05-16 at 9 35 00 AM" src="https://github.com/carbon-design-system/carbon-website/assets/45692622/495b3c48-7c39-4dee-918b-742aeab208a3">

[FlashSystem](https://www.ibm.com/flashsystem)
- Examples throughout their many pages

<img width="1700" alt="Screenshot 2023-05-16 at 9 36 03 AM" src="https://github.com/carbon-design-system/carbon-website/assets/45692622/07f2210e-34d9-4201-996b-765eddade59b">
<img width="1700" alt="Screenshot 2023-05-16 at 9 36 31 AM" src="https://github.com/carbon-design-system/carbon-website/assets/45692622/7bc332f1-0967-4d1e-a4ad-b8b9bf55379f">
<img width="1696" alt="Screenshot 2023-05-16 at 9 37 08 AM" src="https://github.com/carbon-design-system/carbon-website/assets/45692622/fa2c165a-18e1-4552-99de-78f5e2c34866">

**Comment by thyhmdo** [comment]
Moving forward with new updates but there are two Unresolved issues after discussing this with @mbgower and the Design team:

**1. Visited vs Unvisited link**
Michael thinks that WCAG lacks clear guidance on distinguishing between visited and unvisited links, which they see as a failure in addressing the [use of color](https://www.w3.org/WAI/WCAG22/Understanding/use-of-color.html) to convey information. He submitted a pull request (PR) last summer to address this issue, but it seems that the proposed change has not been incorporated into the guidance yet. They provide a link to the merged but unpublished change that they had suggested. https://github.com/w3c/wcag/pull/3222/files

---> We may want to keep this the same in the guidance as an option until there's an update on their side

---
**2. Underline or no underlines for all links**
This seems like an issue to enforce the default links on products. We spotted the inconsistency of how Links are often used with Ghost buttons and it's hard to distinguish them when they appear closely on the same page or at the same position. 

On the Accessibility side, It’s possible to distinguish links by colour, if the contrast is strong enough by [G183: Using a contrast ratio of 3:1 with surrounding text and providing additional visual cues on focus for links or controls where color alone is used to identify them](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G183.html). For dangers of a colour-only approach, see the failure technique [F73: Failure of Success Criterion 1.4.1 due to creating links that are not visually evident without color vision](https://www.w3.org/TR/WCAG20-TECHS/F73.html)

---> We will need to add an underline for the focus state but probably leave the default Standalone link as they are at the moment.
